### DISCUSSION TITLE
Using TurboRepo

### DISCUSSION THREAD

**Discussion by johnhooks** [wp-blocks/wp-blocks#1] [Ideas]
@erikyo I don't know if TurboRepo is going to work with the `@wordpress/*` build tools but I'm going to try, if it doesn't work, `pnpm` workspaces might be enough.

**Comment by erikyo** [comment]
this is something to ask to @gigitux, because he told me he was a big fan of monorepo and would certainly be able to advise us. Furthermore, I also know that he was doing some experiments in this regard maybe he has some news

**Reply by johnhooks** [reply]
Yes! Any advice would be great
